requirements.txt

```diff
@@ -1,4 +1,4 @@
-chroma-hnswlib==0.7.0
+chroma-hnswlib @ git+https://github.com/chroma-core/hnswlib@7ebd96e238aad3dded61a1e1a55abf67444b8500
```

Comment:
Once https://github.com/chroma-core/hnswlib/pull/4 lands we can point this at 0.7.1